src/data/markdown/docs/20 jslib/01 jslib/05 aws/01 S3Client.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'S3Client'
+head_title: 'S3Client'
+description: 'S3Client allows interacting with AWS S3 buckets and objects'
+excerpt: 'S3Client class allows interacting with AWS S3 buckets and objects'
+---
+
+S3Client allows interacting with AWS S3's buckets and objects. Namely, it allows the user to list buckets and the objects they contain, as well as download, uploading, and deleting objects. The S3Client operations are blocking, and we recommend reserving their usage to the [init context](/javascript-api/init-context/), or to the [`setup`](/using-k6/test-life-cycle/) and [`teardown`](/using-k6/test-life-cycle/) functions.
```

Comment:
Is it really possible to call these APIs in the init context? I thought we don't allow HTTP calls in init?

Reply:
You're right 🤦🏻